src/store/actions.ts

```diff
@@ -207,8 +207,15 @@ export default {
     const formFields = [...state.orderFormFields, { id: 'contents', type: 'text' }]
     const formData = {
       ...state.order,
-      ...{ contents: JSON.stringify(toCart(state)) },
-      ...{ updateDate: moment().toISOString() }
+      contents: JSON.stringify(toCart(state)),
+      updateDate: moment().toISOString()
+    }
+
+    if (state.context && state.context.context) {
+      Object.assign(formData, {
+        email: state.context.context.email,
```

Comment:
I think you can safely assume that they exist at this point. And that we don't want to create orders without these fields present.

Reply:
Fixed!